tests/sentry/api/test_event_search.py

```diff
@@ -384,12 +384,15 @@ def test_allowed_keys(self):
         ]
 
     def test_invalid_aggregate_column_with_duration_filter(self):
-        with self.assertRaises(InvalidSearchQuery, regex="not a duration column"):
```

Comment:
This regex= was always broken, on 3.6 you can change it to "foo" and it still passes, lol.

If you change it to the correct `self.assertRaisesRegexp(InvalidSearchQuery, expected_regex="foo")`:

```
E   AssertionError: "foo" does not match "avg(stack.colno):
column argument invalid: stack.colno is not a numeric column"
```